pkgs/tools/bluetooth/nrfconnect/default.nix

```diff
@@ -0,0 +1,44 @@
+{ lib
+, fetchurl
+, appimageTools
+}:
```

Comment:
this is missing `segger-jlink` as an input.

Reply:
ah, sorry, the reason was my way of using it. If segger-jlink is in pkgs, everything is fine.